tests/sles4sap/publiccloud/add_server_to_hosts.pm

```diff
@@ -12,12 +12,15 @@ use qesapdeployment;
 
 sub run {
     my ($self, $run_args) = @_;
-    my $instance = $run_args->{my_instance};
-    record_info("$instance");
-
-    my $ibsm_ip = get_required_var('IBSM_IP');
-    $instance->run_ssh_command(cmd => "echo \"$ibsm_ip download.suse.de\" | sudo tee -a /etc/hosts", username => 'cloudadmin');
-    $instance->run_ssh_command(cmd => 'cat /etc/hosts', username => 'cloudadmin');
+    #my $instance = $run_args->{my_instance};
+    foreach my $instance (@{$run_args->{instances}}) {
```

Comment:
We are doing this one as Ansible is not working right?

Does it make sense to add a comment in the code about it?